docs/index.md

```diff
@@ -59,3 +59,13 @@
    methoddocs/types.md
    methoddocs/utils.md
 ```
+
+```{eval-rst}
+.. toctree::
+   :caption: Plugin Python Reference
+   :maxdepth: 1
+
+   methoddocs/ape_accounts.md
```

Comment:
@dtdang See how I am taking `ape-accounts` out of above and making a new section here for `Plugin Python Reference`? I want us to try and do this for core right now with all the core plugins. I had to make some changes in the core plugings to get it to work though,  had to add `__all__` = in the init, also some other docs files for the auto-modding